src/resolvers/__tests__/service.test.js

```diff
@@ -342,5 +344,650 @@ describe('Service', () => {
         }
       })
     })
+
+    // Tests for the associated legacy services order-options
+    test('only retrieve one OO filtered from assoc details', async () => {
```

Comment:
I think these tests could be organized a little better, add a couple of `describe` blocks. 'When the services are requested with a parent collection query', 'When the services are requested as the top level query'

Reply:
Agree thank you!